Quantum-Control-Applications/Superconducting/Multiple Flux Tunable Transmons/Use Case 2 - Two Qubit Randomized Benchmarking/README.md

```diff
@@ -0,0 +1,238 @@
+# Two-Qubit Randomized Benchmarking
+
+Author: Maximilian Zanner
+
+*Important note: The code in this folder has not been tested in an actual experiment. The code serves as an example and is written for a specifically tailored setup and software environment. When adapting the code to run on your device, make sure to adjust the relevant functions and parameters and do not hesitate to contact QM Customer Success!*
+
+## Introduction
+Two-Qubit Randomized Benchmarking (RB) has become a popular protocol that allows to experimentally quantify the performance of a quantum processor by applying sequences of randomly sampled Clifford gates and measuring the average error rate. Due to its universality it has been implemented in various qubit platforms such as trapped-ions [^1], NMR [^2], spin [^3] and superconducting qubits [^4]. Two-Qubit RB can be challenging to implement with state-of-the-art control electronics because of the necessity to sample from a large amount of Clifford gates corresponding to the Two-Qubit Clifford group. This Clifford group consists of 11520 operations [^4] and contains the single qubit Clifford operations (576), the CNOT-like class (5184), the iSWAP-like class (5184) and the SWAP-like class (576). In this use-case example we introduce a possible implementation on the OPX+ using the current version (2023, June) of the generic *TwoQubitRb* class. The implementation exploits the [*baking tool*](https://github.com/qua-platform/py-qua-tools/blob/main/qualang_tools/bakery/README.md) in order to generate and invert the sequence before running the QUA program. The QUA program then uses the [*Input Stream*](https://docs.quantum-machines.co/0.1/qm-qua-sdk/docs/Guides/features/?h=declare_input_stream#input-streams) feature to randomly load and execute the pre-compiled sequences. An updated version of the *TwoQubitRb* class can be found in the [*py-qua-tools*](https://github.com/qua-platform/py-qua-tools) repository.
```

Comment:
this is not correct as written. We do not use the baking tool to precompile the sequences, only to precompile the individual Cliffords. The then send via the input stream a string of Clifford indices which terminate in the inverse. The inverse is calculated in python.